en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+<td>
+
+Application name owner.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>count</strong></td>
+<td valign="top"><a href="#int">Int</a>!</td>
+<td></td>
+</tr>
+</tbody>
+</table>
+
+### SuccessSummary
+
+Provides successful(2xx response code) usage summary across all the APIs.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>summary</strong></td>
+<td valign="top">[<a href="#successvalue">SuccessValue</a>]!</td>
+<td>
+
+List containing successful usage values. Returns an empty array if no data available.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Granularity value used for data retrieval.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### SuccessValue
+
+Represents successful request count(2xx response code) for a single timestamp. Timestamp is calculated according to the
+provided granularity value.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>timeSpan</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Represents a timestamp value calculated according to the used granularity value. For an example if the used
+granularity value is '1d', timeSpan will represents start of the day calculated adhering to the timezone provided in
+the TimeFilter.
+Ex: timeSpan: '2021-06-21T00:00:00.0000000+05:30'
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>requestCount</strong></td>
+<td valign="top"><a href="#int">Int</a>!</td>
+<td>
+
+Represents successful request count.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### TopAPIByAlertCount
+
+Represents a top API by alert count.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiId</strong></td>
+<td valign="top"><a href="#id">ID</a>!</td>
+<td>
+
+API Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>count</strong></td>
+<td valign="top"><a href="#int">Int</a>!</td>
+<td>
+
+Alert count.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### TopAPIsByAlertCount
+
+Represents top APIs by alert count.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>usage</strong></td>
+<td valign="top">[<a href="#topapibyalertcount">TopAPIByAlertCount</a>]</td>
+<td>
+
+List containing top APIs by alert count. Returns an empty array if no data available.
```

Comment:
Start the first sentence with "A" or "The" as relevant, and add "is" after "data".